### PR TITLE
reorder ekl files

### PR DESCRIPTION
**Pull request by jpeerz**

### CODE REVIEW

File: puppet6/environments/production/modules/orcid_elk/manifests/bconfig.pp [hunk 1]
```diff
@@ -0,0 +1,18 @@
+class orcid_elk::bconfig {
```
Comment (amontenegro) [inline puppet6/environments/production/modules/orcid_elk/manifests/bconfig.pp]: Hey @jpeerz who uses the bconfig?

Comment (jpeerz) [reply puppet6/environments/production/modules/orcid_elk/manifests/bconfig.pp]: it's going to be used by web nodes

Comment (amontenegro) [reply puppet6/environments/production/modules/orcid_elk/manifests/bconfig.pp]: ok good! then it looks ok to go.

**Comment by jpeerz** [comment]
[Review and Document server logging puppet configuration.](https://trello.com/c/zcIOo4OC/19-review-and-document-server-logging-puppet-configuration)

**Pull request merged**